### PR TITLE
Update new host creation docs

### PR DESCRIPTION
**Pull request by matthew-mcallister**
I'll leave this PR open in case our puppet structure changes soon.

### CODE REVIEW

File: ocfweb/docs/docs/staff/procedures/new-host.md [hunk 1]
```diff
@@ -9,10 +9,51 @@ use hostnames of the form `hozer-{60..89}` and their corresponding IP addresses
...
+
+Only do these if a server with this hostname has never existed before (or if
+it's been long enough that some of these steps have never been done before).
+Unfortunately, these steps tend to change a lot as our infrastructure evolves.
+
+## Step 1.1. Add the LDAP entry
+
+On supernova, `kinit $USER/admin ldap-add-host <hostname> <ip>`. If creating a
+staff VM, also do `kinit $USER/admin ldapvi cn=<hostname>` and set the `type`
+attribute to `staffvm`.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: desktop needs a change too

File: ocfweb/docs/docs/staff/procedures/new-host.md [hunk 2]
```diff
@@ -9,10 +9,51 @@ use hostnames of the form `hozer-{60..89}` and their corresponding IP addresses
...
+
+## Step 1.2. Add the DNS record
+
+Clone the [DNS repo][github-dns] from GitHub, run `build-zones`, and push a
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: run `make`

File: ocfweb/docs/docs/staff/procedures/new-host.md [hunk 3]
```diff
@@ -9,10 +9,51 @@ use hostnames of the form `hozer-{60..89}` and their corresponding IP addresses
...
+Run `dns-reconcile` and request the needed DNS changes from the University
+hostmaster.
+
+[github-dns]: https://github.com/ocf/dns
+
+## Step 1.3. Add node config to Puppet
+
+Make a commit to the [Puppet repo][github-puppet] which adds a file
+`hieradata/nodes/<hostname.yaml>` for the new host. Follow the example of
+another server or staff VM's `host.yaml` file.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: only necessary if you need special config for the node, in general this is not necessary

File: ocfweb/docs/docs/staff/procedures/new-host.md [hunk 4]
```diff
@@ -31,7 +72,6 @@ and run `makevm --help`. A typical invocation looks something like:
 
     makevm -m 4096 -c 2 -s 15 arsenic 169.229.226.47
 
-
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: accidental delete?

File: ocfweb/docs/docs/staff/procedures/new-host.md (hunk 1 above)
Comment (matthew-mcallister) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: Can you think of anything else that needs to be changed for desktops? I planned to make another commit for that.

Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: otherwise these docs look fine

File: ocfweb/docs/docs/staff/procedures/new-host.md (hunk 4 above)
Comment (matthew-mcallister) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: For consistency, actually: two newlines between steps, one newline otherwise.

Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: we're pretty inconsistent about it

**Comment by matthew-mcallister** [comment]
I hope the line spacing is now satisfactory.

**Comment by chriskuehl** [comment]
lol
